packages/commerce-sdk-react/src/hooks/ShopperSearch/useShopperSearchProducts.ts

```diff
@@ -0,0 +1,16 @@
+import {ProductSearchResult} from 'commerce-sdk-isomorphic'
+import {ShopperSearchProductParams} from './types'
+import {QueryResponse} from '../../types'
+
+const useShopperSearchProducts = (
```

Comment:
naming nit: Should we name it "useShopperSearchProductSearch" to be consistent with the endpoint name?

https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-search?meta=productSearch

Reply:
+1 all names for these hooks should come from the RAML definitions.

Reply:
Just poked around, all GET endpoints are of the format `getThing` except `productSearch`... 🤔

Reply:
thanks, done